project/src/main/scala/licenses/frontend/SbtLicenses.scala

```diff
@@ -85,7 +85,7 @@ object SbtLicenses {
         val diagnostics =
           if (component.licenseReport.licenses.isEmpty)
             Seq(
-              Diagnostic.Error(
+              Diagnostic.Warning(
```

Comment:
I am not sure I want to be an approval of changes into licensing tool. I don't have any friendly relation with the tool, but maybe due to that I'd rather stay as far as possible.